packages/v4/patternfly-docs/pages/home.js

```diff
@@ -10,269 +24,279 @@ import {
   Title,
   TextContent,
   Text,
-  TextVariants
-} from '@patternfly/react-core';
-import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
-import GithubIcon from '@patternfly/react-icons/dist/esm/icons/github-icon';
-import orb from '../images/pf4-logo.svg';
-import accessible from '../images/accessible.svg';
-import modular from '../images/modular.svg';
-import scale from '../images/scale.svg';
-import consistent from '../images/consistent.svg';
-import open from '../images/open.svg';
-import { AutoLinkHeader } from '@patternfly/documentation-framework/components/autoLinkHeader/autoLinkHeader';
-import './home.css';
+  TextVariants,
+} from "@patternfly/react-core";
+import ArrowRightIcon from "@patternfly/react-icons/dist/esm/icons/arrow-right-icon";
+import GithubIcon from "@patternfly/react-icons/dist/esm/icons/github-icon";
+import ExternalLinkAltIcon from "@patternfly/react-icons/dist/esm/icons/external-link-alt-icon";
+import lightshowcase from "../images/light-showcase.svg";
+import pfLogo from "../images/PF5-IconLogo-color.svg";
+import {AutoLinkHeader} from "@patternfly/documentation-framework/components/autoLinkHeader/autoLinkHeader";
+import "./home.css";
+import featuredPostsData from "./featured-posts-data.json";
 
-const aboutPatternFly = `Hello, we're PatternFly.`;
-const patternFlyPrinciples = 'Flyer fundamentals';
+const patternFlyPrinciples = "Flyer fundamentals";
+
+const cardData = [
+  {
+    title: "Read our design tokens guide",
+    body: "We swapped global CSS variables for design tokens for v5",
+    link: "",
+    layout: "withSubtitle",
+  },
```

Comment:
```suggestion

```
From what I know, design tokens were delayed, so we can remove this card (or point to something else if needed and anyone has other ideas)

Reply:
maybe could point to office hours or something?

Reply:
I like that. @nicolethoen what do you think?

Reply:
Great idea @edonehoo, updated!